docs/download/downloading_data.md

```diff
@@ -0,0 +1,261 @@
+# Downloading Data
+
+## Downloading data from Collaboratory repository
+
+Academic research cloud infrastructure built to house ICGC data.
+
+Repository information can be found [here](https://docs.icgc.org/download/repositories/#collaboratory)
+
+#### Prerequisites
+
+1.  [Apply for DACO access](https://github.com/icgc-dcc/dcc-docs/blob/download-doc-update/docs/download/data-access.md#apply-for-access-to-controlled-data)
+2.  Download and install score-client software [Tarball](/software/download/#score-client), [Docker](https://hub.docker.com/r/overture/score/) (Download/configuration instructions [here](https://docs.icgc.org/download/guide/#installation-of-the-score-client))
+
+<span style="color:blue">NOTE: We are currently actively copying controlled data from [EGA](https://ega-archive.org) to Collaboratory so it can be available to download from either repository. This is an on-going process, so we do not currently have all the EGA datasets in Collaboratory yet.</span>
```

Comment:
Consider this change: `so it can be available to download from either repository` => `so some files originated from EGA may be available to download from either EGA or Collaboratory repository`

Reply:
Thanks Junjun - I re-worded it as you suggested